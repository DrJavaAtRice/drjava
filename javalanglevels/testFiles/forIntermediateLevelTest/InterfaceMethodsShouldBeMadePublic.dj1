//Here, we see that the methods of interfaces are always made public.
interface MyInterfaceYeah {
  boolean equals(Object o);
}