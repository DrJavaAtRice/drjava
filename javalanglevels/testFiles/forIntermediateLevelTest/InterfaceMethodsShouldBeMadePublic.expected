// Language Level Converter line number map: dj*->java. Entries: 4
//     1->3         2->4         3->5         4->6    
//Here, we see that the methods of interfaces are always made public.
interface MyInterfaceYeah {
  public boolean equals(final Object o);
}
